### DISCUSSION TITLE
Number of image in a post

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-checklists#708] [Feature Requests]
Add a new checklist to limit number of image in a post

**Comment by rizaardiyanto1412** [comment]
This week I received another 2 request on this one: https://secure.helpscout.net/conversation/2668943289/224789?viewId=6932143 (I forgot the other one)

@stevejburge @richaferry Do you guys think it's worth to move this idea from Discussions to GitHub issue?

**Comment by stevejburge** [comment]
@rizaardiyanto1412 Sure, go for it. Let's make this a Pro feature

**Comment by stevejburge** [comment]
Solved in a recent update
